src/client/components/ShoppingCart/RemoveFromShoppingCart.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { removeFromShoppingCart } from '../../containers/ShoppingCartPage/ShoppingCartPage.Container';
+import './ShoppingCart.styles.css';
+
+function RemoveFromShoppingCart(props) {
+  const { product, shoppingCart, setShoppingCart } = props;
+
+  const removeProductFromShoppingcart = () => {
+    removeFromShoppingCart(product, shoppingCart, setShoppingCart);
+  };
+
+  return (
+    <div>
+      <div>
+        <button
+          className="close-button"
+          onClick={removeProductFromShoppingcart}
+          type="button"
+        >
+          X
+        </button>
+      </div>
+      <div className="discounted_price">
+        <div
+          style={{
+            display: props.product.discount !== 0 ? 'inline-block' : 'none',
+            'text-decoration': 'line-through',
+          }}
+        >
+          {props.product.currency} {props.product.price}
+        </div>
+        <div
+          style={{
+            display: props.product.discount === 0 ? 'inline-block' : 'none',
+          }}
+        >
+          {product.currency} {product.price}
+        </div>
+        <div
+          style={{
+            display: product.discount !== 0 ? 'inline-block' : 'none',
+          }}
+        >
+          &nbsp;&nbsp;{product.currency}{' '}
+          {Math.round(product.price * (1 - product.discount / 100))}
+        </div>
```

Comment:
We have special offer products now. I think you need to modify once the offer products available in the landing page.

Reply:
use the offer products and offer products list.